webtool/gradle/common/src/commonMain/kotlin/Pilot.kt

```diff
@@ -47,9 +47,14 @@ data class HouseholdGroup(
     val hasHeatPump_r: Double,
     val hasChargePoint_r: Double,
     val hasHomeBattery_r: Double,
-
-    /**Jaarlijks gemiddeld basisverbruik zonder warmtepomp, elektrische voertuigen en zonnepanelen */
+     /**Jaarlijks gemiddeld basisverbruik zonder warmtepomp, elektrische voertuigen en zonnepanelen */
     val annualBaseConsumptionAvg_kWh: Double,
+    // Cost values
+    val costsPer_kWh: Double,
+    val buy_ct: Double,
+    val income_r: Double,
+    val writingPeriod_y: Double,
+    val additionalCosts_cty: Double,
```

Comment:
This field should be in the the root object.

We can call it `bufferPrice_EUR` like in the document "Kostprijs+ methodiek V2.docx"

Reply:
Add "Buffer Price" in the configuration with attribute "bufferPrice_eurpkWh" in the Pilot Object.